values.schema.json

```diff
@@ -61,6 +61,37 @@
           }
         }
       ]
+    },
+    "resources": {
+      "type": "object",
```

Comment:
First of all, provided schema is invalid, plz check https://json-schema.org/.
Second, let's keep things simplier and declare both resources property as an objects, without defining its structure. Just provide a links to the k8s docs will be enough.